include/usb/usb_device.h

```diff
@@ -74,6 +74,14 @@ extern "C" {
 
 #define MAX_PACKET_SIZE0    64        /**< maximum packet size for EP 0 */
 
+#define USB_MPS_MAX_CTRL	64        /**< maximum packet size for EP 0 */
```

Comment:
I think that more readable is `USB_MAX_CTRL_MPS`

Reply:
okay